client/src/main/scala/spatutorial/client/components/DeskRecsChart.scala

```diff
@@ -44,17 +48,19 @@ object DeskRecsChart extends AirportConfig{
                     val potCrunchResult: Pot[CrunchResult] = queueWorkload._1
                     //todo this seems to be at the wrong level as we've passed in a map, only to reach out a thing we're dependent on
```

Comment:
comment stands, are we passing all the data down, only to have the individual components pluck out the bits they want?